src/main/java/org/gridsuite/study/server/repository/sensianalysis/nonevacuatedenergy/NonEvacuatedEnergyGeneratorsLimitEntity.java

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
why do we call it generator capping in the front and limit in the back ?

Reply:
Renamed